hypothesis-python/docs/community.rst

```diff
@@ -15,21 +13,20 @@ applies in all Hypothesis community spaces.
 
 If you would like to cite Hypothesis, please consider :gh-file:`our suggested citation <CITATION.cff>`.
```

Comment:
I notice that https://zenodo.org/records/7866447 (the software citation reference) isn't updated for new versions for the last two years. Something missing in the build pipeline?

Reply:
sounds like it - open an issue?  At least we have a fairly precise place to look for the change!